webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityBody.vue

```diff
@@ -32,6 +32,11 @@ export default {
     body: null,
   }),
   computed: {
+    bodyElement() {
+      return {
+        template: `<div>${this.body}</div>` || '',
```

Comment:
Use the same function here to sanitize manually the body without directive: 
https://github.com/Meeds-io/commons/blob/9564910c74dae791d1f87c1ca5155ddb735451eb/commons-extension-webapp/src/main/webapp/WEB-INF/gatein-resources.xml#L428-L445

BTW, this function should be added inside a centralized function that is used here and that can be used elsewhere, such as here: 
https://github.com/Meeds-io/social/blob/6134dad651b2a4b119c14871a1d5fa4d36de5f08/webapp/portlet/src/main/webapp/vue-apps/common/js/Utils.js#L2

Reply:
Ok, `HTMLSanitizer` is already used to purify HTML, thus I close this as resolved.